dtmi/adlink/I_pi_smarc_imx8mp_plus-1.json

```diff
@@ -0,0 +1,13 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:ADLINK:I_Pi_SMARC_IMX8MP_plus;1",
```

Comment:
the filename should be lowercase. I'd suggest you to use the dmr-client to import these files with the expected folder/filename.